### DISCUSSION TITLE
How to install ungoogled-cromium on ParrotOS?

### DISCUSSION THREAD

**Discussion by VladRDV** [Eloston/ungoogled-chromium#2570] [Q&A] [answered]
I've been trying for days to get ungoogled-chromium on my ParrotOS 5( Debian family), but nothing seems to work

I tried building it myself as described in the guide provided in the repo, build is successful, but browser won't even launch(instructions provided are for vanilla Debian, but usually apps for Debian just work). Then I tried AppImage made for debian from the website, same thing.

**Comment by networkException** [comment, marked as answer]
I'm not familiar with ParrotOS but the flatpak will probably work.

**Comment by VladRDV** [comment]
Just in case someone else comes to the same situation as I

flatpak version of ungoogled chromium  works like a charm, installing flatpak can be done with apt
